packages/babel-helper-replace-supers/src/index.js

```diff
@@ -158,9 +158,12 @@ export default class ReplaceSupers {
     property: Object,
     value: Object,
     isComputed: boolean,
+    loose: boolean = false,
   ): Object {
     return t.callExpression(this.file.addHelper("set"), [
-      getPrototypeOfExpression(this.getObjectRef(), this.isStatic),
+      loose
+        ? t.memberExpression(this.superRef, t.identifier("prototype"))
+        : getPrototypeOfExpression(this.getObjectRef(), this.isStatic),
```

Comment:
Seems like much of this PR is about making loose mode even looser. I'm wondering what the motivation is here--do you have any benchmarks showing that this version is faster? Or is this about consistency with other places where loose mode takes shortcuts?